src/WordPress/Hooks.php

```diff
@@ -12,6 +12,7 @@ class Hooks
     protected $api;
     protected $config;
     protected $dataStore;
+	protected $integrationContext;
```

Comment:
can you switch your editors config from tabs to spaces. Since everything is already space let's continue doing spaces.